Dockerfile

```diff
@@ -11,8 +11,8 @@ ENV AIRFLOW_VERSION 1.10.1
 ENV AIRFLOW_HOME /usr/local/airflow
 ENV SLUGIFY_USES_TEXT_UNIDECODE=yes
 
-# Pipe-tools
-ENV PIPE_TOOLS_VERSION v1.0.0
+#Airflow-gfw
+ENV AIRFLOW_GFW_VERSION v0.0.1-dev3
```

Comment:
@smpiano I think we should keep PIPE_TOOLS for now. So we should have both here:
```
ENV AIRFLOW_GFW_VERSION v0.0.1
ENV PIPE_TOOLS_VERSION v1.0.0
```

That way is easier to migrate. We should be able to remove `ENV PIPE_TOOLS_VERSION v1.0.0` next week, but in the meantime I can work with this in production.